mutation_calling/fusion/fusion-inspector/Dockerfile

```diff
@@ -0,0 +1,50 @@
+# ProTECT Dockerfile for STAR-Fusion
+#
+# Build with: sudo docker build --force-rm --no-cache -t jpfeil/fusion-inspector:1.0.1 - < Dockerfile
+# Run with sudo docker run -v <IO_folder>:/data jpfeil/fusion-inspector:1.0.1 <parameters>
+
+# Use pimmuno
+FROM aarjunrao/pimmuno
+
+MAINTAINER Jacob Pfeil, jpfeil@ucsc.edu
+
+RUN apt-get update --fix-missing && \
+    apt-get install -y dpkg zlib1g-dev gzip perl libdb-dev build-essential make
+
+RUN echo "deb http://ppa.launchpad.net/webupd8team/java/ubuntu xenial main" | tee /etc/apt/sources.list.d/webupd8team-java.list && \
+    echo "deb-src http://ppa.launchpad.net/webupd8team/java/ubuntu xenial main" | tee -a /etc/apt/sources.list.d/webupd8team-java.list && \
+    apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys EEA14886 && \
+    apt-get update && \
+    echo oracle-java8-installer shared/accepted-oracle-license-v1-1 select true | /usr/bin/debconf-set-selections && \
+    apt-get install -y oracle-java8-installer
+
+RUN cpan App::cpanminus && cpanm Set::IntervalTree && cpanm DB_File && cpanm URI
+
+WORKDIR /opt
+
+# Download Wrappers 
+RUN git clone https://github.com/jpfeil/toil_docker_wrappers.git /opt/docker_wrappers
+
+RUN wget -qO- https://github.com/FusionInspector/FusionInspector/releases/download/v1.0.1/FusionInspector-v1.0.1.FULL.tar.gz | tar xz && \
+    wget -qO- https://github.com/alexdobin/STAR/archive/2.5.2b.tar.gz | tar xz && \
+    wget -qO- https://github.com/trinityrnaseq/trinityrnaseq/archive/Trinity-v2.4.0.tar.gz | tar xz && \
+    cd trinityrnaseq-Trinity-v2.4.0 && make && cd /opt && \
+    wget -qO- http://research-pub.gene.com/gmap/src/gmap-gsnap-2016-11-07.tar.gz | tar xz && \
+    cd gmap-2016-11-07 && ./configure && make && make check && make install
+
+# FusionInspector requires samtools 1.3+        
+RUN wget -qO- https://downloads.sourceforge.net/project/samtools/samtools/1.3.1/samtools-1.3.1.tar.bz2 | tar xj && \
+    wget -qO- https://downloads.sourceforge.net/project/samtools/samtools/1.3.1/htslib-1.3.1.tar.bz2 | tar xj 
+
+RUN cd samtools* && make prefix=/usr/local/bin && make install
```

Comment:
make a samtools 1.3+ it can't be too different from the samtools that exists. Add it to this PR as a separate commit!